bentoml/_internal/utils/validation.py

```diff
@@ -2,46 +2,27 @@
 
 from bentoml.exceptions import InvalidArgument
 
-dns1123_label_fmt = "[_a-z0-9]([-_a-z0-9]*[_a-z0-9])?"
-dns1123_subdomain_max_length = 253
-dns1123_subdomain_max_length_error_msg = (
-    "a tag must be less than {dns1123_subdomain_max_length} characters in length"
+tag_fmt = "[a-z0-9]([-._a-z0-9]*[a-z0-9])?"
+tag_max_length = 63
+tag_max_length_error_msg = (
+    "a tag's name or version must be at most {tag_max_length} characters in length"
 )
-dns1123_subdomain_fmt = dns1123_label_fmt + "(\\." + dns1123_label_fmt + ")*"
-dns1123_subdomain_error_msg = "a tag must consist of alphanumeric characters, '_', '-', or '.', and must not start or end with '-'"
-dns1123_subdomain_regex = re.compile(f"^{dns1123_subdomain_fmt}$")
+tag_invalid_error_msg = "a tag's name or version must consist of alphanumeric characters, '_', '-', or '.', and must start and end with an alphanumeric character"
+tag_regex = re.compile(f"^{store_name_fmt}$")
```

Comment:
saw it from the unit test logs -> store_name_fmt not defined

Reply:
Yep, forgot to actually fix this...